pkg/commands/git_commands/branch_loader_test.go

```diff
@@ -12,13 +14,18 @@ func TestObtainBranch(t *testing.T) {
 	type scenario struct {
 		testName       string
 		input          []string
+		withRecency    bool
```

Comment:
This should be called `storeCommitDateAsRecency`, it's one of the inputs to the function we're testing. (Changed in 5f272433fabd)